src/components/ui/CreateFormDropdown.tsx

```diff
@@ -1,104 +1,76 @@
-  { label: "Create a new form for Applicants", type: FormType.Applicant },
-]
-
-interface CreateFormDropdownProps {
-  applicationFormExists: boolean
-}
-
-export default function CreateFormDropdown({
-  applicationFormExists,
-}: CreateFormDropdownProps) {
-  const navigate = useNavigate()
-  const dispatch = useDispatch()
-  const [cookies] = useCookies([Cookie.jwt])
-
-  const [createForm] = useCreateFormMutation()
-
-  const potentialForms = applicationFormExists
-    ? menuItems
-    : [
-        {
-          label: "Create a new Application form",
-          type: FormType.Application,
-        },
-        ...menuItems,
-      ]
-
-  const handleCreateForm = async (type: FormType) => {
-    const nextFormTitle = `${type} form name...`
-    const requestBody: {
-      name: string
-      type: FormType
-      startDate?: string
-      endDate?: string
-      stages?: { name: string }[]
-    } = { name: nextFormTitle, type }
-
-    if (type === FormType.Application) {
-      requestBody.startDate = dayjs().add(1, "day").toISOString()
-      requestBody.endDate = dayjs().add(2, "day").toISOString()
-      requestBody.stages = [{ name: "Application" }]
-    }
-
-    try {
-      const result = await createForm({ jwt: cookies.jwt, body: requestBody })
-
-      if (result.error) {
-        throw result.error
-      }
-
-      navigate(`/forms/${result?.data?._id}`)
-    } catch (error) {
-      const { message } = getErrorInfo(error)
-      handleShowAlert(dispatch, {
-        type: AlertType.Error,
-        message,
-      })
-    }
-  }
-
-  return (
-    <Menu>
-      <MenuButton className="flex items-center gap-2 bg-primary-dark text-white px-4 py-3 rounded">
-        <PlusIcon />
-        Create form
-      </MenuButton>
-
-      <MenuItems
-        transition
-        anchor="bottom end"
-        className="origin-top-right rounded-lg border border-white/5 bg-white text-sm text-gray-500 font-medium transition duration-100 ease-out [--anchor-gap:var(--spacing-1)] focus:outline-none data-[closed]:scale-95 data-[closed]:opacity-0 custom-shadow"
-      >
-        {potentialForms.map((item, index) => (
-          <MenuItem key={index}>
-            <button
-              className={classNames(
-                "group flex w-full items-center py-3 px-5 data-[focus]:bg-primary-dark data-[focus]:text-white border-b",
-                { "border-none": index === potentialForms.length - 1 },
-              )}
-              onClick={() => {
-                handleCreateForm(item.type)
-              }}
-            >
-              {item.label}
-            </button>
-          </MenuItem>
-        ))}
-      </MenuItems>
-    </Menu>
-  )
-}
+import { Menu, MenuButton, MenuItem, MenuItems } from "@headlessui/react"
+import { PlusIcon } from "@radix-ui/react-icons"
+import { FormType } from "../../utils/types"
+import classNames from "classnames"
+import CreateForm from "../modals/CreateForm"

```

Comment:
I would recommend in the future setting up a prettier and Eslint configuration and checks for prettier and Eslint in the CI (GitHub actions)